index.html

```diff
@@ -32,7 +32,8 @@
           <header>
             <h1 class="Name">Javed Shahbaz</h1>
           </header>
-          <p class="headLine">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+          <p class="headLine">Hello! I am a software developer!
```

Comment:
- [ ] Please Ensure that this section matches the Figma Design 👇🏾 
   Consider the following: 
   - The font size and font-family
   - Spacing - margin/padding
   - The `see project` button is touch the bottom edge.

Your Design
![image](https://user-images.githubusercontent.com/84156714/221373954-673811b7-e0e0-47db-bb03-7ceed8b3c4d4.png)

Figma Design
![image](https://user-images.githubusercontent.com/84156714/221374058-3dd30610-1b8f-4bbb-97b7-9c324fa21dc7.png)

Reply:
Fixed.